aws_cloudtrail_policies/aws_cloudtrail_s3_bucket_public.py

```diff
@@ -1,18 +1,18 @@
-from panther_oss_helpers import resource_lookup
 from panther_base_helpers import deep_get
+from panther_oss_helpers import resource_lookup
 
 BAD_PERMISSIONS = {
-    'http://acs.amazonaws.com/groups/global/AuthenticatedUsers',
-    'http://acs.amazonaws.com/groups/global/AllUsers',
+    "http://acs.amazonaws.com/groups/global/AuthenticatedUsers",
+    "http://acs.amazonaws.com/groups/global/AllUsers",
 }
 
 
 def policy(resource):
-    bucket_arn = 'arn:aws:s3:::' + resource['S3BucketName']
+    bucket_arn = "arn:aws:s3:::" + resource["S3BucketName"]
     bucket = resource_lookup(bucket_arn)
 
-    for grant in bucket['Grants'] or []:
-        if deep_get(grant, 'Grantee', 'URI') in BAD_PERMISSIONS:
+    for grant in bucket["Grants"] or []:
```

Comment:
same question as above, although here its a nested value (did we also want to update these to use `.get()` / `deep_get()` ?)